lib/docker/DockerClient.js

```diff
@@ -22,7 +22,7 @@ class DockerClient extends Docker {
    */
   constructor(opts) {
     super(_.defaults({}, opts, config.docker, {
-      timeout: 60 * 1000
+      timeout: 170 * 1000
```

Comment:
can we move the magic number to a `constants.js`? Also a comment at the place where this is defined as to how this number was arrived at would help in future.